bin/hdfcoinc/pycbc_multiifo_combine_statmap

```diff
@@ -0,0 +1,208 @@
+#!/bin/env python
+""" Apply a naive mass binning, assuming that each bin is fully independent, which 
+is a conservative estimate. This clusters to find the most significant foreground, but
+leaves the background triggers alone. """
+
+import h5py, numpy, argparse, logging, pycbc, pycbc.events, lal
+import pycbc.version
+from scipy.special import comb
+import matplotlib as mpl
+mpl.use('agg')
+import matplotlib.pyplot as plt
+
+def com(f, files, group):
+    """ Combine the same column from multiple files and save to a third"""
+    f[group] = numpy.concatenate([fi[group][:] if group in fi else \
+                                   numpy.array([], dtype=numpy.uint32) for fi in files])
+
+def all_keys_set(files):
+    key_set = []
+    for fi in files:
+        for k in fi.keys():
+            if isinstance(fi[k], h5py.Dataset):
+                key_set.append(k)
+            else:
+                key_set += get_all_subkeys(fi,k)
+    return set(key_set)
+
+def get_all_subkeys(fi,k):
+    subkey_set = []
+    for sk in fi[k].keys():
+        if isinstance(fi[k+'/'+sk], h5py.Dataset):
+            subkey_set.append(k+'/'+sk)
+        else:
+            subkey_set += get_all_subkeys(fi,k+'/'+sk)
+    # this will return an empty list if there is no dataset or subgroup within the group
+    #   - so avoids infinite iterations
+    return subkey_set
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--version", action="version", version=pycbc.version.git_verbose_msg)
+parser.add_argument('--verbose', action='store_true')
+parser.add_argument('--statmap-files', nargs='+',
+                    help="List of coinc files to be redistributed")
+parser.add_argument('--cluster-window', type=float)
+parser.add_argument('--output-file', help="name of output file")
+parser.add_argument('--ifos',nargs='+', 
+                    help="list of interferometers in the input files - note that this " \
```

Comment:
not sure about this requirement - what happens if it's not in the 'right' order?  

.. is it necessary to specify this on the command line, can't the code work it out from the input files?